tests/phpunit/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -221,6 +221,7 @@ private function get_order_details( $order ) {
 			'order_id'       => $order_id,
 			'order_key'      => $order_key,
 			'payment_type'   => 'single',
+			'signature'      => sprintf( '%d:%s', $order->get_id(), md5( $order->get_order_key() . $order->get_id() . $order->get_customer_id() ) ),
```

Comment:
```suggestion
			'signature'      => sprintf( '%d:%s', $order->get_id(), md5( implode( '-', [ absint( $order->get_id() ), $order->get_order_key(), $order->get_customer_id(), $amount ] ) ) ),
```
In order to match [the function](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/fix/webhook-processing-when-intent-id-is-missing/includes/abstracts/abstract-wc-stripe-payment-gateway.php#L2337-L2346) and fix the failed tests.